src/ui/screens/home/controller/ControllerImage.qml

```diff
@@ -176,9 +174,7 @@ Rectangle {
         },
         State {
             name: "noSignalOn"
-            when: (VPNController.state === VPNController.StateOn ||
-                   VPNController.state === VPNController.StateSilentSwitching) &&
-                VPNConnectionHealth.stability === VPNConnectionHealth.NoSignal
+            when: VPNController.isVPNActive && VPNController.state != VPNController.StateSilentSwitching && VPNConnectionHealth.stability === VPNConnectionHealth.NoSignal
```

Comment:
why did we go from needing to be on OR silent switching to needing to be on AND NOT silent switching